### ISSUE TITLE
There seems to be an issue with "jed flip" ing a residue

### ISSUE THREAD

**Issue by pemsley** [stuartjamesmcnicholas/Moorhen#336]
There seems to be an issue with "jed flip" ing a residue which one has split into an alternate conformer. Simple to reproduce ("Get monomer" (NUT) -> Make alternate conformation->Jed-flip). At first I hoped it was just because we were not adding the alternate conformer ID into the atom CID given to the command, but I have now pushed a change that addresses that scenario, and the atom spec now includes the alternate conf

**Comment by pemsley** [comment]
What happens? Nothing moves? Whereas when you don't have the alt conf, it does work? Is that the case?

**Comment by pemsley** [comment]
Wait, what? Am I talking to myself?

**Comment by FilomenoSanchez** [comment]
> Wait, what? Am I talking to myself?

Yes you were the first to create this card in the project. I am not sure what you meant at that time.
